CHANGELOG.md

```diff
@@ -13,6 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+# 0.11.0 - 2020-09-23
+
+### Changed
+
+- BUGFIX: The SearchBox styling has been fixed and the component no longer shows up as ForwardRef.
+- Vanilla has been updated to 2.19.0.
+- ModularTable has updated the `columns.accessor` prop to be either a string or component.
```

Comment:
Technically it's not a component, but a function that calculates the value based on row object.

```suggestion
- ModularTable has updated the `columns.accessor` prop to be either a string or getter function.
```

Reply:
Ohhh, I see. I'll update.